Jenkinsfile.treecompose

```diff
@@ -57,13 +70,19 @@ node(NODE) {
             return
         }
 
-        stage("Compose Tree") {
-            sh "coreos-assembler --repo=${repo} ${manifest}"
-            sh "ostree --repo=${repo} rev-parse ${ref} > commit.txt"
+        // Note that we don't keep history in the ostree repo
```

Comment:
Don't we still need to `ostree prune` though to drop out old objects from the previous commit?

Reply:
Yep, fixed thanks :arrow_down: